### DISCUSSION TITLE
Disabling gameplay HUD does not hide multiplayer leaderboard

### DISCUSSION THREAD

**Discussion by Nyxa** [ppy/osu#29808] [Q&A] [answered]
### Type

Game behaviour

### Bug description

Currently, setting the HUD overlay visibility mode to "Never" or "Hide during gameplay" will still show the leaderboard in multiplayer, even if "Always show gameplay leaderboard" is disabled.

Even if this is intended, this is very unintuitive. I think everyone would expect that hiding the HUD would hide all HUD elements, including the leaderboard. If there really is some important reason that the leaderboard must always be displayed, then at the very least some in-game explanation (or at least a heads-up that it's intended) should be given.

### Screenshots or videos

![image](https://github.com/user-attachments/assets/4f304e41-ddca-452a-93cc-e6bc7895535a)

### Version

2024.906.2-lazer

### Logs

[compressed-logs.zip](https://github.com/user-attachments/files/16925768/compressed-logs.zip)

**Comment by peppy** [comment]
You can use tab to toggle the leaderboard's display mode.

**Reply by bdach** [reply]
Not in multiplayer you can't.

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/issues/27850
